cmd/agent/common/common_windows.go

```diff
@@ -173,6 +174,25 @@ func ImportRegistryConfig() error {
 	} else {
 		log.Debug("hostname not found, not setting")
 	}
+	if val, _, err = k.GetStringValue("hostname"); err == nil {
+		config.Datadog.Set("hostname", val)
+		log.Debugf("Setting hostname %s", val)
+	} else {
+		log.Debug("hostname not found, not setting")
```

Comment:
I would say: `hostname not found in registry: using default value`
What do you think ?